eprihlaska/views.py

```diff
@@ -781,15 +791,15 @@ def admin_list():
 @app.route('/admin/view/<id>')
 @require_remote_user
 def admin_view(id):
-    app = ApplicationForm.query.filter_by(id=id).first()
```

Comment:
@Adman After your very nice comment I took the chance to refactor this everywhere where it made sense.

Thanks again!